src/modules/my-positions/selectors/my-positions.js

```diff
@@ -11,6 +10,11 @@ export default function () {
 export const selectPositionsMarkets = createSelector(
   selectMarkets,
   selectAccountPositionsState,
-  selectAllUserOpenOrderMarkets,
-  (markets, positions, marketIDs) => (markets || []).filter(market => Object.keys(positions || {}).find(positionMarketID => market.id === positionMarketID) || (marketIDs || {}).find(marketID => market.id === marketID))
+  (markets, positions) => {
+    if (!markets || !positions) {
+      return []
+    }
+    (markets || []).filter(market => Object.keys(positions || {})
```

Comment:
Need return here, accidently pushed this

Reply:
positions is keyed array of 0,1 (buy|sell) of arrays of positions with market  id.